File: python/sum_of_odd.py
def sum_of_odd(n:list) -> int:
    if len(n) <= 2:
        raise ValueError("invalid array")
    sum = 0
    for i in n:
        if i % 2 == 1:
            sum += i

    return sum


def main():
    arr = [1,2,3,4,5,6]

    sum = sum_of_odd(arr)
    print(f'The sum of the odd number is: {sum}')


main()

File: typescript/src/sum_of_odd.ts
const sumOfOdd = (list:number[]):number => {
    if(list.length <= 1){
        return list[0];
    }

    let sum = 0;

    for(let i of list){
        if (i % 2 === 1){
            sum += i;
        }
    }

    return sum
}

const list_ = [1,2,3,4,5,6];
const value = sumOfOdd(list_)

console.log(`Sum of the even number is: ${value}`);